pypeerassets/card_parsers.py

```diff
@@ -1,15 +1,27 @@
 '''parse cards according to deck issue mode'''
 
-from .pautils import exponent_to_amount
+from typing import Callable, Optional
 
 
-def none_parser(cards):
+def amount_to_exponent(amount: float, number_of_decimals: int) -> int:
```

Comment:
This belongs in pautils, as it's used by some other tooling from there too. It's a utility function.